phpcs.xml.dist

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<ruleset name="Custom Post Type UI">
+    <description>The code standard for Custom Post Type UI plugin.</description>
+
+    <!-- What to scan -->
+    <file>.</file>
+    <exclude-pattern>apigen/</exclude-pattern>
+    <exclude-pattern>assets/</exclude-pattern>
+    <exclude-pattern>css/</exclude-pattern>
+    <exclude-pattern>images/</exclude-pattern>
+    <exclude-pattern>js/</exclude-pattern>
+    <exclude-pattern>node_modules/</exclude-pattern>
+    <exclude-pattern>vendor/</exclude-pattern>
+
+    <!-- How to scan -->
+    <arg value="sp"/> <!-- Show sniff and progress -->
+    <arg name="colors" />
+    <arg name="extensions" value="php"/>
+    <!--<arg name="report" value="full"/>-->
+    <!--<arg name="report" value="summary"/>-->
+    <arg name="report" value="source"/>
+
+    <!-- Rules: Good use of variables -->
+    <rule ref="VariableAnalysis"/>
+
+    <!-- Rules: Check PHP version compatibility -->
+    <config name="testVersion" value="5.2-"/>
+    <rule ref="PHPCompatibility"/>
+
+    <!-- Rules: WordPress Coding Standards -->
+    <config name="minimum_supported_wp_version" value="4.6"/>
+    <rule ref="WordPress">
+        <exclude name="WordPress.VIP"/>
+    </rule>
+    <rule ref="WordPress.Files.FileName">
+        <properties>
+            <property name="strict_class_file_names" value="false"/>
+        </properties>
+        <exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
+    </rule>
+    <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+        <properties>
+            <property name="prefixes" type="array" value="cptui"/>
+        </properties>
+    </rule>
+    <rule ref="WordPress.WP.I18n">
+        <properties>
+            <property name="text_domain" type="array" value="custom-post-type-ui"/>
+        </properties>
+    </rule>
+    <rule ref="WordPress.WhiteSpace.ControlStructureSpacing">
+        <properties>
+            <property name="blank_line_check" value="true"/>
+        </properties>
+    </rule>
+
+</ruleset>
```

Comment:
Travis-CI is failing with this one: https://cloudup.com/cf2bbNd3Jla

Apparently a dependency needs PHP7.1+

Reply:
`composer why-not php "~7.0.0"` shows it's `doctrine/instantiator`, which I happen to know comes from phpunit.

This is the reason to remove `composer.lock` from the repo, so that Travis, or someone with only PHP 5.6 on their system, can install whatever dependencies work with their version of PHP.

If you wish, I can switch to PHP 7.0, `composer update`, and commit the updated lock file, though better would be to remove it altogether.

@tw2113 Which do you want?

Reply:
would not having the lock file in there allow for a lower php version for the doctrine/instantiator part? If yes, then i'm game.